pennylane_lightning/core/_serialize.py

```diff
@@ -261,16 +263,16 @@ def _pauli_sentence(self, observable, wires_map: dict = None):
     # pylint: disable=protected-access
     def _ob(self, observable, wires_map: dict = None):
         """Serialize a :class:`pennylane.operation.Observable` into an Observable."""
-        if isinstance(observable, Tensor):
-            return self._tensor_ob(observable, wires_map)
-        if observable.name == "Hamiltonian":
-            return self._hamiltonian(observable, wires_map)
-        if observable.name == "SparseHamiltonian":
-            return self._sparse_hamiltonian(observable, wires_map)
         if isinstance(observable, (PauliX, PauliY, PauliZ, Identity, Hadamard)):
             return self._named_obs(observable, wires_map)
         if observable._pauli_rep is not None:
             return self._pauli_sentence(observable._pauli_rep, wires_map)
+        if isinstance(observable, (Tensor, Prod)):
+            return self._tensor_ob(observable, wires_map)
+        if observable.name in ("Hamiltonian", "LinearCombination"):
```

Comment:
Could this just be an `isinstance` check?

Reply:
Definitely can be.

Reply:
Let's do this when we stumble upon such cases.